tests/codemods/test_request_verify.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from codemodder.codemods.requests_verify import RequestsVerify
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+
+REQUEST_FUNCS = ["get", "post", "request"]
+
+
+class TestRequestsVerify(BaseSemgrepCodemodTest):
```

Comment:
These are good tests, but just for my own sanity could we add the following:
1. A test with multiple kwargs
2. A test where `verify` is not the first kwarg
(I see this is in the integration test but it would be good for unit tests as well).